pkg/resource/logging-config/alloy-logging-config.go

```diff
@@ -0,0 +1,13 @@
+package loggingconfig
+
+import (
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+)
+
+// GenerateAlloyLoggingConfig returns a configmap for
+// the logging extra-config
+// This is a plaholder function, in case we need to populate the configmap
+// for Alloy in the future.
+func GenerateAlloyLoggingConfig(lc loggedcluster.Interface) (string, error) {
```

Comment:
Is there no way to only put secrets in the secret file as we did for promtail and have the config part here?

Reply:
There's no obvious way to achieve that, but I will attempt something in this direction

Reply:
I moved the general config into the logging-config configmap, set the secret in logging-secret and pass them over to alloy using `envFrom`.